tests/contexts/index.js

```diff
@@ -0,0 +1,39 @@
+if(require.extensions) {
+  require.extensions['.jsonld'] = require.extensions['.json'];
```

Comment:
This feature is deprecated:
https://nodejs.org/api/modules.html#modules_require_extensions

Reply:
thanks removing it